### PR TITLE
build: plugin 6.3.5, gradle 8.0.3, jakarta validation

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: mqttv3/src/main/java/io/micronaut/mqtt/v3/config/MqttClientConfigurationProperties.java [hunk 1]
```diff
@@ -35,7 +34,6 @@
  * @since 1.0.0
  */
 @ConfigurationProperties("mqtt.client")
-@Context
```
Comment (sdelamo) [inline mqttv3/src/main/java/io/micronaut/mqtt/v3/config/MqttClientConfigurationProperties.java:38]: why did you remove `@Context`?

Comment (wetted) [reply mqttv3/src/main/java/io/micronaut/mqtt/v3/config/MqttClientConfigurationProperties.java:38]: The test fails with it, because it inits too early. I was a getting a validation failure. The v5 version of it is not `@Context`. Is there a better way to fix that?

**Review by sdelamo — APPROVED** [review]
